src/assets/js/navbar.js

```diff
@@ -0,0 +1,10 @@
+
+console.log("called")
+const button = document.getElementById("collapse-button");
+const navbar = document.getElementById("collapseExample");
+console.log(button)
+console.log(navbar)
```

Comment:
```suggestion
const button = document.getElementById("collapse-button");
const navbar = document.getElementById("collapseExample");
```
Remove the debug logs.

Reply:
Done